include/experimental/__p0009_bits/extents.hpp

```diff
@@ -306,13 +329,72 @@ class extents
       std::conditional_t<N==rank_dynamic(),
         detail::__construct_psa_from_dynamic_exts_array_tag_t<0>,
         detail::__construct_psa_from_all_exts_array_tag_t>(),
-        std::array<SizeType,N>{exts}
+      std::array<SizeType,N>{exts}
 #if defined(_MDSPAN_USE_ATTRIBUTE_NO_UNIQUE_ADDRESS)
       }
 #else
       }})
 #endif
-  { }
+  {
+    /* TODO: precondition check
+     * If N != rank_dynamic() is true, exts[r] equals Er for each r for which Er is a static extent, and
+     * either
+     *   - N is zero, or
+     *   - exts[r] is nonnegative and is a representable value of type size_type for all rank index r
+     */
+  }
+
+#if MDSPAN_HAS_CXX_20
```

Comment:
Is the point of testing `MDSPAN_HAS_CXX_20` just to work around an NVCC bug?  If so, then would you consider just replacing this with `#ifdef __NVCC__` for now, until we can do better testing for specifically supported NVCC versions?  If not, then would you consider using specifically relevant feature test macros, like `__cpp_concepts`?

Reply:
span is C++20 ... so the constructor doesn't work if we don't have that :-)